coordinator/persistence-dse-6.8/src/main/java/io/stargate/db/dse/impl/Conversion.java

```diff
@@ -681,10 +681,20 @@ private static MD5Digest getResultMetadataId(ResultSet.ResultMetadata metadata)
     return Conversion.toExternal(metadata.getResultMetadataId());
   }
 
+  // Dse and stargate enums are different
+  private static final Map<Integer, Result.Flag> globalTablesSpec = new HashMap<>();
```

Comment:
init in static block and make unmodifiable..

Reply:
Wouldn't be able to use Map.of() as these are still in Java 8. Need to be done in 2 lines, imo it's not needed as its a private map.